tests/tensorflow/nightly/targets.jsonnet

```diff
@@ -24,13 +24,13 @@ local wmt = import 'tf-wmt-wmt14_translate.libsonnet';
 
 // Add new models here
 std.flattenArrays([
-  //  dlrm.configs,
+  dlrm.configs,
```

Comment:
@chandrasekhard2 why are we uncommenting these tests?

Reply:
Yes, we're uncommenting all the nightly tests